packages/core/src/products/listing/redux/selectors.js

```diff
@@ -743,3 +749,51 @@ export const getHierarchicalFacetsWithChildren = createSelector(
     });
   },
 );
+
+/**
+ * Returns the listing facets.
+ *
+ * @function
+ *
+ * @param {object} state - Application state.
+ *
+ * @returns {object} Listing facets object.
+ */
+export const getListingFacets = state =>
+  getListingFacetsState(state.listingFacets);
+
+/**
+ * Returns the listing facets error.
+ *
+ * @function
+ *
+ * @param {object} state    - Application state.
+ *
+ * @returns {object}        - Listing Facets error.
+ */
+export const getListingFacetsError = state =>
+  getListingFacetsState(state).error;
+
+/**
+ * Returns the result of the listing facets.
+ *
+ * @function
+ *
+ * @param {object} state    - Application state.
+ *
+ * @returns {Array}         - Array of listing facets.
+ */
+export const getListingFacetsResult = state =>
```

Comment:
We should also provide a way to fetch a specific facet. 

We can expose a selector that calls the `getEntity` selector and retrieve a facet by a given ID. Keep in mind that this ID structure should take into consideration the type of facet (for now we will need for `size by category` but in the future we can use this selector to provide another type of facet).